ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+### Prerequisites
+
+We thank you for reporting a bug or feature request and helping us to make the SeqAn library better.
+We would like you to check the following points before continuing with the submission as often
+some items might have been solved already. 
```

Comment:
some language :nail_care: 
> Thank you for creating an issue and helping us improve the SeqAn library! Before you continue, please check the following list items to avoid the most common pitfalls and help us reply more quickly and accurately: